lib/kennel/models/slo.rb

```diff
@@ -35,7 +35,9 @@ def build_json
           type: type
         )
 
-        if v = query
+        if type == 'time_slice' && v = sliSpecification
+          data[:sliSpecification] = v
```

Comment:
does it only support sliSpecification or is setting query also fine ?
... because if it needs sliSpecification we should do this:
```suggestion
        if type == 'time_slice'
          data[:sliSpecification] = sli_specification
```